src/RetrievesHelper.ts

```diff
@@ -0,0 +1,34 @@
+import { R4 } from '@ahryman40k/ts-fhir-types';
+import { MeasureBundleHelpers, CalculatorTypes, ELMTypes, RetrievesFinder } from 'fqm-execution';
+
+/**
+ * Get all retrieves used in the library
+ *
+ * @param measureBundle Bundle with a MeasureResource and all necessary data for execution.
+ * @returns DataTypeQuery array of all retrieves output
```

Comment:
We should update this docstring since we are no longer returning a DataTypeQuery array